app/assets/javascripts/directives/user_avatar.js

```diff
@@ -0,0 +1,10 @@
+angular.module("OpenHq").directive("userAvatar", function() {
+    return {
+        restrict: "E",
+        scope: {
+          user: '=',
+          size: '=',
+        },
+        template: JST['templates/users/avatar'],
```

Comment:
should we make a sub folder in templates called directives just to keep them all separate?